bus-mapping/src/rpc.rs

```diff
@@ -100,6 +100,17 @@ impl<P: JsonRpcClient> GethClient<P> {
             .await
             .map_err(|e| Error::JSONRpcError(e.into()))
     }
+    /// ..
+    pub async fn get_tx_by_hash(&self, hash: H256) -> Result<Transaction, Error> {
```

Comment:
this rpc call targets on L2geth node right ?

Reply:
both l1 and l2 work. This rpc is intended to be used to mock prove single tx.